packages/server-wallet/src/protocols/state.ts

```diff
@@ -106,6 +106,13 @@ export function directFundingStatus(
   myParticipant: Participant,
   fundingStrategy: FundingStrategy
 ): FundingStatus | undefined {
+  if (fundingStrategy === 'Fake') {
+    if (!supported) return 'Uncategorized';
+    if (supported.turnNum < 3) return 'ReadyToFund';
+    if (supported.isFinal) return 'Defunded';
```

Comment:
Is this mean to be correct? It is incorrect, in the case when the latest supported state is final, but there is no conclusion proof.

Reply:
I'll just remove this branch for now — no code relies on the "Defunded" status of a closed fake funded channel at this moment.

Reply:
Now, a fake-funded channel with a conclusion proof is listed as `'Funded'`. Is that intended?